docusaurus/docs/develop/envoy/introduction.md

```diff
@@ -56,11 +55,13 @@ title: Introduction
 
 2. Run `make init_envoy` to create all the required config files
```

Comment:
Can you revert this? I pushed changes to make the bullet points easier to read and added an info below.

Note that I'm leaving comments rather than editing the README because I want you to see/review the changes/improvements/edits I made.